cli/cmd/certificateChain.go

```diff
@@ -24,19 +24,20 @@ func newCertificateChain() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			hostName := args[0]
-			return cliCertificateChain(hostName, certFilename, eraConfig, insecureEra)
+			return cliCertificateChain(hostName, certFilename, eraConfig, insecureEra, acceptedTcbLevels)
 		},
 		SilenceUsage: true,
 	}
 
 	cmd.Flags().StringVarP(&certFilename, "output", "o", "marblerunChainCA.crt", "File to save the certificate to")
+	cmd.PersistentFlags().StringSliceVar(&acceptedTcbLevels, "accepted-tcb-levels", []string{"UpToDate"}, "Coma seperated list of user accepted TCB levels (e.g. ConfigurationNeeded,ConfigurationAndSWHardeningNeeded)")
```

Comment:
```suggestion
```
Don't need to define the flag again since it is already defined by the parent command.